core/DSGToolsProcessingAlgs/Algs/ValidationAlgs/topologicalAreaDouglasSimplificationAlgorithm.py

```diff
@@ -108,40 +111,50 @@ def processAlgorithm(self, parameters, context, feedback):
         layerHandler = LayerHandler()
         algRunner = AlgRunner()
 
-        inputLyrList = self.parameterAsLayerList(parameters, self.INPUTLAYERS, context)
+        inputLyrList = self.parameterAsLayerList(
+            parameters, self.INPUTLAYERS, context)
         if inputLyrList is None or inputLyrList == []:
-            raise QgsProcessingException(self.invalidSourceError(parameters, self.INPUTLAYERS))
+            raise QgsProcessingException(self.invalidSourceError(
+                parameters, self.INPUTLAYERS))
         onlySelected = self.parameterAsBool(parameters, self.SELECTED, context)
         snap = self.parameterAsDouble(parameters, self.SNAP, context)
-        threshold = self.parameterAsDouble(parameters, self.DOUGLASPARAMETER, context)
+        threshold = self.parameterAsDouble(
+            parameters, self.DOUGLASPARAMETER, context)
         minArea = self.parameterAsDouble(parameters, self.MINAREA, context)
-        self.prepareFlagSink(parameters, inputLyrList[0], QgsWkbTypes.MultiPolygon, context)
+        self.prepareFlagSink(
+            parameters, inputLyrList[0], QgsWkbTypes.MultiPolygon, context)
 
 
         multiStepFeedback = QgsProcessingMultiStepFeedback(3, feedback)
         multiStepFeedback.setCurrentStep(0)
         multiStepFeedback.pushInfo(self.tr('Building unified layer...'))
-        coverage = layerHandler.createAndPopulateUnifiedVectorLayer(inputLyrList, geomType=QgsWkbTypes.MultiPolygon, onlySelected = onlySelected, feedback=multiStepFeedback)
+        coverage = layerHandler.createAndPopulateUnifiedVectorLayer(
+            inputLyrList, geomType=QgsWkbTypes.MultiPolygon, 
+            onlySelected=onlySelected, feedback=multiStepFeedback)
 
         multiStepFeedback.setCurrentStep(1)
         multiStepFeedback.pushInfo(self.tr('Running clean on unified layer...'))
-        simplifiedCoverage, error = algRunner.runDouglasSimplification(coverage, \
-                                                    threshold,\
-                                                    context, \
-                                                    returnError=True, \
-                                                    snap=snap, \
-                                                    minArea=minArea,
-                                                    feedback=multiStepFeedback)
-
+        simplifiedCoverage, error = algRunner.runDouglasSimplification(
+            coverage,
+            threshold,
+            context,
+            returnError=True,
+            snap=snap,
+            minArea=minArea,
+            feedback=multiStepFeedback)
```

Comment:
Similar ao anterior, quando separa cada argumento em uma linha, alinho o parêntese com o início da linha que provocou:
```python
simplifiedCoverage, error = algRunner.runDouglasSimplification(
    coverage,
    threshold,
    context,
    returnError=True,
    context,
    snap=snap,
    returnError=True,
    feedback=multiStepFeedback
)
```

```